src/tools/interop/idt/requirements.txt

```diff
@@ -0,0 +1,3 @@
+zeroconf==0.74.0
+bleak==0.20.2
```

Comment:
It would be nice to upgrade to 0.21.0. The Release notes mention that they added support for Python 3.12, which would be nice to have.

After you pin to specific versions, it usually sticks until someone intentionally is trying to fight through a specific issue a few years later and they don't have any context for why specific versions were used and it adds regression risk when trying to migrate.

Reply:
Thanks! I've updated the version.